### DISCUSSION TITLE
Dataset Doubt

### DISCUSSION THREAD

**Discussion by DEADSAW** [DEADSAW/DecisionHub#4] [Q&A]
What are the attributes we are looking to put in the dataset?

**Comment by vix2704** [comment]
List of all attributes(columns) in dataset: unique_id, name, age ,gender, marital status, education, annual income,credit score, employment status, debt-to-income ratio, loan amount, loan tenure, collateral, number of dependents
------------------------------------------------------------------------------------------------------------------------------------------------
attributes to be checked initially: 1. Annual income 2. Credit score 3. Loan amount
attributes that can be added: age, dependency(Y/N), Collateral(Y/N), Employment status(Y/N)

**Reply by DEADSAW** [reply]
Got it.
